front-page.php

```diff
@@ -0,0 +1,24 @@
+<?php
+get_header();
+
+$splash_text = IGV_get_option('_igv_splash_text');
+$splash_text_en = IGV_get_option('_igv_splash_text_en');
+
```

Comment:
Possibly but already existing in the previous WP like this. How does tagline deal with language and q-translate?